typeSummary.txt

```diff
@@ -21049,72 +21052,67 @@ class Microsoft.Graph.Group : Microsoft.Graph.DirectoryObject
   property TransitiveMembers : Microsoft.Graph.IGroupTransitiveMembersCollectionWithReferencesPage
   property UnseenCount : System.Nullable`1[[System.Int32]]
   property Visibility : System.String
-class Microsoft.Graph.GroupAcceptedSendersCollectionPage : Microsoft.Graph.CollectionPage`1[[Microsoft.Graph.DirectoryObject]]
-  property NextPageRequest : Microsoft.Graph.IGroupAcceptedSendersCollectionRequest
-  method InitializeNextPageRequest
-    return type System.Void
-    param client : Microsoft.Graph.IBaseClient
-    param nextPageLinkString : System.String
-class Microsoft.Graph.GroupAcceptedSendersCollectionRequest : Microsoft.Graph.BaseRequest
+class Microsoft.Graph.GroupAcceptedSendersCollectionReferencesRequest : Microsoft.Graph.BaseRequest
   method AddAsync
-    return type System.Threading.Tasks.Task`1[[Microsoft.Graph.DirectoryObject]]
+    return type System.Threading.Tasks.Task
     param directoryObject : Microsoft.Graph.DirectoryObject
   method AddAsync
-    return type System.Threading.Tasks.Task`1[[Microsoft.Graph.DirectoryObject]]
+    return type System.Threading.Tasks.Task
     param directoryObject : Microsoft.Graph.DirectoryObject
     param cancellationToken : System.Threading.CancellationToken
+class Microsoft.Graph.GroupAcceptedSendersCollectionReferencesRequestBuilder : Microsoft.Graph.BaseRequestBuilder
+  method Request
+    return type Microsoft.Graph.IGroupAcceptedSendersCollectionReferencesRequest
+  method Request
+    return type Microsoft.Graph.IGroupAcceptedSendersCollectionReferencesRequest
+    param options : System.Collections.Generic.IEnumerable`1[[Microsoft.Graph.Option]]
+class Microsoft.Graph.GroupAcceptedSendersCollectionWithReferencesPage : Microsoft.Graph.CollectionPage`1[[Microsoft.Graph.DirectoryObject]]
+  property NextPageRequest : Microsoft.Graph.IGroupAcceptedSendersCollectionWithReferencesRequest
+  method InitializeNextPageRequest
+    return type System.Void
+    param client : Microsoft.Graph.IBaseClient
+    param nextPageLinkString : System.String
+class Microsoft.Graph.GroupAcceptedSendersCollectionWithReferencesRequest : Microsoft.Graph.BaseRequest
   method Expand
-    return type Microsoft.Graph.IGroupAcceptedSendersCollectionRequest
+    return type Microsoft.Graph.IGroupAcceptedSendersCollectionWithReferencesRequest
```

Comment:
@zengin same comment as  https://github.com/microsoftgraph/msgraph-sdk-java/pull/630 this is technically a breaking change but I'm guessing it's broken without that change?

Reply:
Yes, this is one of those changes. Technically breaking, but it is a bug fix.

If you think that it is a huge impact to the customers, we can push it to major releases (which are coming soon for both Java and C#).

Reply:
If the methods were not working before we're only fixing something that nobody could have had a dependency on previously because it was broken. Which means I'm fine shipping as a minor.
If somebody could've taken a dependency on it because it was working in some cases, I think we need to wait for another major version.

Reply:
This is from an email thread with the service folks:

> FYI, I tried AcceptedSender api on microsoft graph, they use references.

I have also tested without a `/$ref` to see if anybody could have taken a dependency because some OData URLs work without it too. In this particular instance, the API doesn't work without a `/$ref`.

So I think we can fix it without a major release.

Reply:
thanks for taking the time to double check!